### PR TITLE
feature/146

### PR DESCRIPTION
**Pull request by hkukreja**
Implements #146 

<|<- META INFO (DO NOT MODIFY CONTENTS MANUALLY) ->|>
*BLOCKED BY*
No Issues

*BLOCKING*

- [ ] #146

<|<- META INFO END ->|>

### CODE REVIEW

**Review by l2silver — COMMENTED** [review]
So this looks pretty interesting. I'll need to talk it over with the board to see how they feel, and then we'll see about merging this in. Thanks!

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (dnephin) [inline LICENSE.md:1]: This file is not markdown, so I guess the filename should be `LICENSE` or `LICENSE.txt`.

File: web-server/volunteer_toronto_be/urls.py [hunk 2]
```diff
@@ -34,8 +34,7 @@
             "responses of the API endpoints.",
         terms_of_service="",
         contact=openapi.Contact(email="info@diglit.com"),
-        license=openapi.License(name="SAP Individual Contributor License Agreement"),
-            #https://gist.github.com/CLAassistant/bd1ea8ec8aa0357414e8
+        license=openapi.License(name="BSD License"),
```
Comment (dnephin) [inline web-server/volunteer_toronto_be/urls.py:37]:
```suggestion
        license=openapi.License(name="GPLv3"),
```

**Review by dnephin — COMMENTED** [review]
Thank you for the PR and for raising the issue! 

Reading over https://www.gnu.org/licenses/gpl-howto.html, I believe in order to correctly apply the license we also need a `NOTICE` or `NOTICE.txt` file with contents like this:


    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <https://www.gnu.org/licenses/>.


It also talks about applying a header to every file, but I've never really been a fan of that approach, and it's not clear to me if that is strictly necessary or if it's simply done out of an abundance of caution.

There are potentially a couple other places we could add the license information as well, but maybe it's not strictly necessary:
* in the [package.json](https://github.com/diglit/volunteer-toronto-volunteer-drivers-app/blob/develop/package.json) - https://docs.npmjs.com/cli/v7/configuring-npm/package-json#license
* in [web-server/pyproject.toml](https://github.com/diglit/volunteer-toronto-volunteer-drivers-app/blob/develop/web-server/pyproject.toml) - https://python-poetry.org/docs/pyproject/#license

**Review by l2silver — APPROVED** [review]

**Pull request merged**